Cubical/Foundations/Prelude.agda

```diff
@@ -213,6 +213,22 @@ _∎ : (x : A) → x ≡ x
 _ ∎ = refl
 
 
+-- BEGIN: what is the idiom to do this kind of implicational reasoning?
+
+infix  3 _◼
+infixr 2 _→⟨_⟩_ _→⟨⟩_
+
+_→⟨_⟩_ : ∀{ℓ ℓ' ℓ''} {Q : Type ℓ'} {R : Type ℓ''} → (P : Type ℓ) → (P → Q) → (Q → R) → (P → R)
+_ →⟨ p→q ⟩ q→r = λ x → q→r (p→q x)
+
+_→⟨⟩_ : ∀{ℓ ℓ'} {Q : Type ℓ'} → (P : Type ℓ) → (P → Q) → (P → Q)
+_ →⟨⟩ p→q = p→q
+
+_◼ : ∀{ℓ} (A : Type ℓ) → A → A
+_ ◼ = λ x → x
+
+-- END
```

Comment:
I would like to use this in the proof of `+-reflects-<ʳ` in `Cubical.Data.Int.Properties` but I found it useful for developing proofs in a little bit more verbose way.

Might this be something to add to the standard library? Otherwise I would remove it and collapse the proof.